src/Microsoft.NET.Sdk.Functions.Generator/FunctionJsonConverter.cs

```diff
@@ -173,36 +175,78 @@ private bool TryGenerateFunctionJsons()
                 AssemblyResolver = resolver
             };
 
-            var module = ModuleDefinition.ReadModule(_assemblyPath, readerParams);
+            loadContext.Resolving += (context, assemblyName) =>
+            {
+                Assembly result = null;
+
+                // First, check the assembly root dir. Assemblies copied with the app always wins.
+                var path = Path.Combine(assemblyRoot, assemblyName.Name + ".dll");
+                if (File.Exists(path))
+                {
+                    result = context.LoadFromAssemblyPath(path);
+
+                    if (result != null)
+                        return result;
+                }
+
+                // Then, check if the assembly is already loaded into the default context.
+                // This is typically the case for framework assemblies like System.Private.Corelib.dll.
+                if (AssemblyLoadContext.Default.Assemblies.FirstOrDefault(a => a.GetName().Equals(assemblyName)) is Assembly existing)
+                {
+                    path = existing.Location;
+                    result = context.LoadFromAssemblyPath(path);
+
+                    if (result != null)
```

Comment:
We have a coding convention to always use `{ }`, even for single-liners like this (and with each bracket on its own line). Can you update your `if` statements with that convention?

Reply:
Done. Though do let me know if I overlooked any.